app/controllers/v1/mixtapes_controller.rb

```diff
@@ -0,0 +1,14 @@
+class V1::MixtapesController < V1::ApplicationController
+
+  def index
+
+  end
+
+  def create()
```

Comment:
No `()` needed.

However, using a `create` leads me to think that the Mixtape is actually being creates as a resource.

I'd expect to get a status code :201 (created) back.

Maybe this is just a GET show?